packages/client/src/interface/Home.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, CategoryButton, Header, styled, Column, Typography } from "@revolt/ui";
+import { IS_DEV, IS_REVOLT, useClient } from "@revolt/client";
+import { useTranslation } from "@revolt/i18n";
+import { useNavigate } from "@revolt/routing";
+
+import { Match, Switch } from "solid-js";
+
+import { BiRegularMoney, BiSolidCompass, BiSolidCog, BiSolidMegaphone, BiSolidPlusCircle, BiSolidRightArrowCircle } from "solid-icons/bi";
+import { modalController } from "@revolt/modal";
+
+/**
+ * Base layout of the home page (i.e. the header/background)
+ */
+const Base = styled("div")`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  background: ${({ theme }) => theme!.colours["background-200"]};
+`;
+
+/**
+ * Layout of the content as a whole
+ */
+const Content = styled("div")`
+  width: fit-content;
+  margin: auto;
+`
+
+/**
+ * Layout of the buttons
+ */
+const Buttons = styled("div")`
+  display: flex;
+`;
+
+export function HomePage() {
+    const t = useTranslation();
+    const navigate = useNavigate();
+    const client = useClient();
+    // check if we're revolt.chat; if so, check if the user is in the Lounge
+    const showLoungeButton = IS_REVOLT;
+    const isInLounge = client.servers.get("01F7ZSBSFHQ8TA81725KQCSDDP") !== undefined;
+
+    return (
+        <Base>
+            <Header palette="primary">Home</Header>
+            <Content>
+                <Typography
+                    style={"text-align: center; margin-bottom: 1em; font-size: 200%;"}
+                    variant="legacy-settings-title">
+                    {t("app.special.modals.onboarding.welcome")}
+                    <br />
+                    <img style="margin-top: 0.5em;" height={80} src="/assets/wide.svg" />
+                </Typography>
+                <Buttons>
+                    <Column style={"margin-right: 0.25em;"}>
```

Comment:
Hard-coded styles throughout code, if really necessary, just wrap the component:
```
const Abc = styled(Column)` margin-right: 0.25em; `;
```
Also, use margin-inline-end or whatever the equivalent is.